tests/integration/repos_test.go

```diff
@@ -8,11 +8,10 @@
 package tests
 
 import (
+	"github.com/google/go-github/github"
 	"net/http"
 	"reflect"
 	"testing"
-
-	"github.com/google/go-github/github"
```

Comment:
All imports have been ordered. The test cases have now RequiredPullRequestReviews also.